chef/data_bags/crowbar/template-provisioner.json

```diff
@@ -9,6 +9,7 @@
       "access_keys": "",
       "shell_prompt": "USER@HOST:CWD SUFFIX",
       "enable_pxe": true,
+      "self_update_url": "",
```

Comment:
:) Well, the default in this case is to use `scc.suse.com` which works. But we could also use a crowbar local default. Which would you prefer?